crates/sui-node/src/lib.rs

```diff
@@ -1034,12 +1034,15 @@ impl SuiNode {
                     )
                 } else {
                     // was a fullnode, still a fullnode
-                    let is_inconsistent = self.check_is_consistent_state(
-                        self.accumulator.clone(),
-                        cur_epoch_store.epoch(),
-                        false,
-                    );
-                    checkpoint_executor.set_inconsistent_state(is_inconsistent);
+                    // Suppress state checks until we are caught up to the end of the epoch storm.
+                    if new_epoch_store.epoch() > 740 {
+                        let is_inconsistent = self.check_is_consistent_state(
```

Comment:
non blocking at all but it'd be great if we can track the time spent in this call for the future.

Reply:
Going to put up a separate PR to do this non-blocking. Assuming we're not blocking reconfig, the time this takes shouldn't matter, as it's actually a temp check. It'll be removed eventually